### PR TITLE
Update timezone of sig-scalability public meetings

### PR DESCRIPTION
**Pull request by mm4tt**

### CODE REVIEW

File: sig-scalability/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ SIG Scalability is responsible for defining and driving scalability goals for Ku
 The [charter](charter.md) defines the scope and governance of the Scalability Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [Thursdays at 17:30 UTC](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (bi-weekly ([upcoming meeting dates](#upcoming-2019-meeting-dates))). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=UTC).
+* Regular SIG Meeting: [Thursdays at 18:30 Warsaw time (CET or CEST)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (bi-weekly ([upcoming meeting dates](#upcoming-2019-meeting-dates))). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:30&tz=Warsaw%20time%20%28CET%20or%20CEST%29).
```
Comment (wojtek-t) [inline sig-scalability/README.md]: The link doesn't work for - I'm in WAW and it shows me that it's 20:30 :)

Comment (mm4tt) [reply sig-scalability/README.md]: Now it should work :)

**Pull request merged**